### PR TITLE
Lint json and yaml files

### PR DESCRIPTION
**Pull request by jeanregisser**
### Description

We unknowingly merged an invalid translation json file in https://github.com/valora-inc/wallet/commit/a13f16b61f8a75f7ecea81b10bd89984da1c8a14#diff-e975964e274dbbd8ca524cb9a258ed3225508b3c6d0c4e2198f3566a78380bf4R1070 (see #1736) which caused the Crowdin sync to fail.

This PR now makes sure `json` and `yaml` files are linted so we don't merge invalid json and yaml files in the future.

Note: the translation file fix was actually [merged earlier](https://github.com/valora-inc/wallet/commit/6eb70ce88710df6854d8472ed8c299e66fb9e215#diff-e975964e274dbbd8ca524cb9a258ed3225508b3c6d0c4e2198f3566a78380bf4R1070), that's why it's not present in the diff here.

### Tested

- `yarn prettify:diff` returns an errors with invalid json/yaml files

### How others should test

N/A

### Related issues

- Discussed in [Slack](https://valora-app.slack.com/archives/C029Z1QMD7B/p1641982482018500).

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/mobile/src/geth/rinkeby.json [hunk 1]
```diff
@@ -1 +0,0 @@
...00000000000000000000000000f0":{"balance":"0x1"},"00000000000000000000000000000000000000f1":{"balance":"0x1"},"00000000000000000000000000000000000000f2":{"balance":"0x1"},"00000000000000000000000000000000000000f3":{"balance":"0x1"},"00000000000000000000000000000000000000f4":{"balance":"0x1"},"00000000000000000000000000000000000000f5":{"balance":"0x1"},"00000000000000000000000000000000000000f6":{"balance":"0x1"},"00000000000000000000000000000000000000f7":{"balance":"0x1"},"00000000000000000000000000000000000000f8":{"balance":"0x1"},"00000000000000000000000000000000000000f9":{"balance":"0x1"},"00000000000000000000000000000000000000fa":{"balance":"0x1"},"00000000000000000000000000000000000000fb":{"balance":"0x1"},"00000000000000000000000000000000000000fc":{"balance":"0x1"},"00000000000000000000000000000000000000fd":{"balance":"0x1"},"00000000000000000000000000000000000000fe":{"balance":"0x1"},"00000000000000000000000000000000000000ff":{"balance":"0x1"},"31b98d14007bdee637298086988a0bbd31184523":{"balance":"0x200000000000000000000000000000000000000000000000000000000000000"}},"number":"0x0","gasUsed":"0x0","parentHash":"0x0000000000000000000000000000000000000000000000000000000000000000"}
```
Comment (kathaypacific) [inline packages/mobile/src/geth/rinkeby.json:1]: just checking we don't need this anymore?

File: packages/mobile/src/fragmentTypes.json [hunk 2]
```diff
@@ -1 +0,0 @@
-{"__schema":{"types":[{"kind":"UNION","name":"Event","possibleTypes":[{"name":"Exchange"},{"name":"Transfer"}]},{"kind":"INTERFACE","name":"TokenTransaction","possibleTypes":[{"name":"TokenTransfer"},{"name":"TokenExchange"}]}]}}
```
Comment (kathaypacific) [inline packages/mobile/src/fragmentTypes.json:1]: and this?

**Review by kathaypacific — APPROVED** [review]
YES 🚀

**Review by gnardini — APPROVED** [review]
Nice!

**Review by jh2oman — APPROVED** [review]
Oh goodie! Thank you 🙏

File: packages/mobile/src/geth/rinkeby.json (hunk 1 above)
Comment (jeanregisser) [reply packages/mobile/src/geth/rinkeby.json:1]: Yep this is an obsolete file.

File: packages/mobile/src/fragmentTypes.json (hunk 2 above)
Comment (jeanregisser) [reply packages/mobile/src/fragmentTypes.json:1]: Yes obsolete too.

**Comment by ValoraQA** [comment]
Hey @MuckT since "How others should test has n/a" hence we are skipping this PR

CC : @jeanregisser

**Pull request merged**